include/avif/avif.h

```diff
@@ -311,8 +311,8 @@ enum
     // This is actually reserved, but libavif uses it as a sentinel value.
     AVIF_COLOR_PRIMARIES_UNKNOWN = 0,
 
-    AVIF_COLOR_PRIMARIES_SRGB = 1,
     AVIF_COLOR_PRIMARIES_BT709 = 1,
+    AVIF_COLOR_PRIMARIES_SRGB = 1,
```

Comment:
Note that I didn't reorder `	AVIF_TRANSFER_CHARACTERISTICS_PQ` and `AVIF_TRANSFER_CHARACTERISTICS_SMPTE2084` for the transfer_characteristics value 16. I think we can make an exception for that constant because PQ is better known than SMPTE ST 2084.